apps/media-viewer/src/Mediaviewer.vue

```diff
@@ -57,6 +57,7 @@
             class="oc-cursor-pointer"
             size="medium"
             name="chevron_left"
+            aria-label="Previous"
```

Comment:
Translate the strings pls. Ref on how it can be done - https://github.com/owncloud/phoenix/blob/0ba693cea50cafdb7e6b4e3f559a70cbd4d04bbe/apps/files/src/components/AllFilesList.vue#L14

Do so for the other two as well pls.

Reply:
@LukasHirt done!